### ISSUE TITLE
<title>

### ISSUE THREAD

**Issue by 1337jck** [cabarius/ToyBox#1090]
### Is your feature request related to a problem?

Feature is not really related to a problem, but rather convenience.

### Describe the solution you'd like

A menu similar to the mod called "Craft Magic Items" from Pathfinder Kingmaker.
Adding different effects and enhancements can be quite chaotic

### Describe alternatives you've considered

A similar mod would be "CraftMaster" by "skyswordkill" but it seems he has stopped working on it and only rudimentary features are implemented

### Additional context

Basically, if you could add a function tab that offers the functionality of "Craft Magic Items" a popular mod from Kingmaker
